sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -113,48 +159,104 @@ public SensitivityAnalysisResult run(Network network,
                                              SensitivityAnalysisParameters parameters,
                                              ComputationManager computationManager,
                                              ReportNode reportNode) {
-            return runAsync(network, workingVariantId, factors, contingencies, variableSets, parameters, computationManager, reportNode).join();
+            return run(network, workingVariantId, factors,
+                new SensitivityAnalysisRunParameters()
+                    .setContingencies(contingencies)
+                    .setVariableSets(variableSets)
+                    .setParameters(parameters)
+                    .setComputationManager(computationManager)
+                    .setReportNode(reportNode));
         }
 
+        /**
+         * @deprecated use {@link #run(Network, String, List, SensitivityAnalysisRunParameters)} instead
+         */
+        @Deprecated(since = "6.9.0", forRemoval = true)
         public SensitivityAnalysisResult run(Network network,
                                              String workingVariantId,
                                              List<SensitivityFactor> factors,
                                              List<Contingency> contingencies,
                                              List<SensitivityVariableSet> variableSets,
                                              SensitivityAnalysisParameters parameters) {
-            return runAsync(network, workingVariantId, factors, contingencies, variableSets, parameters, LocalComputationManager.getDefault(), ReportNode.NO_OP).join();
+            return run(network, workingVariantId, factors,
+                new SensitivityAnalysisRunParameters()
+                    .setContingencies(contingencies)
+                    .setVariableSets(variableSets)
+                    .setParameters(parameters));
         }
 
+        /**
+         * @deprecated use {@link #run(Network, List, SensitivityAnalysisRunParameters)} instead
+         */
+        @Deprecated(since = "6.9.0", forRemoval = true)
         public SensitivityAnalysisResult run(Network network,
                                              List<SensitivityFactor> factors,
                                              List<Contingency> contingencies,
                                              List<SensitivityVariableSet> variableSets,
                                              SensitivityAnalysisParameters parameters) {
-            return run(network, VariantManagerConstants.INITIAL_VARIANT_ID, factors, contingencies, variableSets, parameters);
+            return run(network, network.getVariantManager().getWorkingVariantId(), factors,
```

Comment:
We previously had `VariantManagerConstants.INITIAL_VARIANT_ID` here... which is weird because it is not really intuitive and it is not documented.

I think that this new value makes more sense, but that it should be changed in another PR to highlight this modification.

=>
- In this PR: use `VariantManagerConstants.INITIAL_VARIANT_ID` as default variant ID.
- New PR to create to change this value to network.getVariantManager().getWorkingVariantId()